cli/tests/data/marathon/apps/zero_instance_sleep.json

```diff
@@ -4,6 +4,7 @@
   "cpus": 0.1,
   "mem": 16,
   "instances": 0,
+  "dependencies": [],
```

Comment:
I'll go ahead and add one, but I'm not really interested in testing every aspect of this schema. that would be a lot of tests.